README.md

```diff
@@ -4,51 +4,31 @@
 
 ApiCenter is a repository for all your OpenAPI specifications. You can upload new ones and search them so that finding the one you need is easy.
 
-## Prerequisites
+## How to use
 
-### General
+### Deploy to cloud (heroku)
```

Comment:
I'm not sure if I'd put Heroku so prominently in there. After all, ApiCenter is most likely to be deployed internally, and Heroku is not often used for that.